src/backend/distributed/citus--6.0-18--6.1-1.sql

```diff
@@ -0,0 +1,76 @@
+/* citus--6.0-18--6.1-1.sql */
+
+SET search_path = 'pg_catalog';
+
+ALTER TABLE pg_dist_colocation ADD COLUMN defaultgroup BOOLEAN;
+
+UPDATE pg_dist_colocation SET defaultgroup = TRUE;
+
+DROP INDEX pg_dist_colocation_configuration_index;
+
+CREATE INDEX pg_dist_colocation_configuration_index
+ON pg_dist_colocation USING btree(shardcount, replicationfactor, distributioncolumntype, defaultgroup);
+
+DROP FUNCTION create_distributed_table(regclass, text, citus.distribution_type);
+
+CREATE FUNCTION create_distributed_table(table_name regclass,
+										 distribution_column text,
+										 distribution_type citus.distribution_type DEFAULT 'hash',
+										 colocate_with text DEFAULT 'default')
```

Comment:
Nope, you can have functions with the same name, but different arguments. In that case, they will show up as separate functions in \df as well, which can be very informative since you can specify separate comments.